### ISSUE TITLE
`sudo xl console Qubename` can't connect and doesn't say why not

### ISSUE THREAD

**Issue by straussmaus** [QubesOS/qubes-issues#6973]
[How to file a helpful issue](https://www.qubes-os.org/doc/issue-tracking/)

### Qubes OS release

4.x

### Brief summary

After upgrading a standalone Debian 10 Qube to bullseye, the Qube will not start and `sudo xl console` gives no useful information.

### Steps to reproduce

- Install Debian 10 standalone qube

- upgrade to bullseye, or maybe it stopped halfway and I didn't notice?

- have the following problems

### Expected behavior

Both `guest-Qubename.log` should show meaningful error message and `sudo xl console Qubename` should connect to Qube or explain why it can't

### Actual behavior

An error says to check `/var/log/xen/console/guest-Qubename.log` but this file does not indicate the problem.

This file does show that it is taking a long time to start a network device.

Later this file shows a login prompt.

`qvm-console Qubename` gave `Error: domain 'Qubename' does not exist or is not running`

And `sudo xl console Qubename` connected during the startup process, showing the Network Manager errors in realtime.  But after Qubename fails to start, it gives:

`Qubename is an invalid domain identifier (rc=-6)`

But I have used these commands on this domain before and the name is correct.

So the bug to report is that I can not connect to Qube using `sudo xl console` or `qvm-console` and there is no meaningful error messaging.

Later, I tried connecting `sudo xl console Qubename` while it was starting up and I can log in, but it does not appear to be the correct Qube.  Its hostname is `localhost` not `Qubename` and the files that should be there are not, and anyway it kicks me out after I get the original error in my console saying 

`Domain Qubename has failed to start: Cannot connect to qrexec agent for 60 seconds, see /var/log/xen/console/guest-Qubename.log for details`

I feel like the actual bug here is the error messaging for `sudo xl console` and maybe if we resolve that, I can connect and find the underlying problem.

**Comment by DemiMarie** [comment]
Have you tried `qvm-console-dispvm`?

**Comment by straussmaus** [comment]
Yes.  First time, it tries to connect, I see an xterm pop up for less than one second, and then the command ends and I'm back on the dom0 command line.

So I try it again immediately and I get:

`Error: domain 'Qubename' does not exist or is not running`

(And it's not running because the Qube failed to start.)
